src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -948,44 +982,6 @@ class ActivityResourceIntTest : IntegrationTestBase() {
         }
       }
 
-      @Test
-      fun `should retain the end date if pay rate start date is moved back at same time as rate is changed`() {
```

Comment:
Removed because this test doesn't make sense - it's not possible to move the start date back.